src/components/Signup/Signup.js

```diff
@@ -109,6 +109,7 @@ const Signup = function ({
   const utmCampaign = getParameter(location, 'utm_campaign');
   const utmMedium = getParameter(location, 'utm_medium');
   const utmTerm = getParameter(location, 'utm_term');
+  const gclid = getParameter(location, 'gclid');
 
   const utmCookies = manageUtmCookies(localStorage, utmSource, utmCampaign, utmMedium, utmTerm);
```

Comment:
If I am not wrong, the `gclid` value should be also included in `utmCookies`.

Maybe you can change the signature of `manageUtmCookies()` to make it easy future improvements:

```suggestion
  const gclid = getParameter(location, 'gclid');

  const utmCookies = manageUtmCookies({ localStorage, utmSource, utmCampaign, utmMedium, utmTerm,  gclid });
```